fuzz/ass.dict

```diff
@@ -0,0 +1,191 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# Keywords for dictionary-assisted ASS fuzzing
+# For use with AFL++'s -x and libfuzzer's -dict option
+#
+# Syntax:
+#   Lines starting with # are ignored.
+#   Keywords are listed as name="keyword", with the alphanumeric
+#   identifier name follooed by = being optional.
+#   Inside the quotes " and \ must be escaped by \" and \\ respectively
+#   and non-printable characters must be escaped as \x<hex>.
+
+# Some common font names
+"Arial"
+"DejaVu Sans"
+
+# Various common terminals
+"yes"
+"no"
+"0x"
```

Comment:
I’m actually not sure if `0x` is also accepted as a hexadecimal prefix anywhere.

Reply:
It is (`mystrtoi32`). In colour headers, even `0x0x` is accepted.